conf/runner.yaml

```diff
@@ -691,3 +691,8 @@ smtp:
     #auth:
     #    user: someuser
     #    pass: somepass
+
+slack:
+    username: tls-observatory
+    icon_emoji: ':telescope:'
```

Comment:
If this is the new 'emoji status' thing, is there also an accompany status_text field that should be commented out when unused?